src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1014,14 +1014,13 @@ MavlinkReceiver::handle_message_set_position_target_global_int(mavlink_message_t
 
 		const uint16_t type_mask = target_global_int.type_mask;
 
-		// position
-		if (!(type_mask & (POSITION_TARGET_TYPEMASK_X_IGNORE | POSITION_TARGET_TYPEMASK_Y_IGNORE |
-				   POSITION_TARGET_TYPEMASK_Z_IGNORE))) {
+		// position x,y
+		if (!(type_mask & (POSITION_TARGET_TYPEMASK_X_IGNORE | POSITION_TARGET_TYPEMASK_Y_IGNORE))) {
```

Comment:
Has the case where position setpoints are sent, but either xy or z is ignored without velocity setpoints?